README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] There should be a ```.gitignore``` file listing test.md file and node_modules directory in the repo, please add a this into the gitignore file